src/main/java/com/bookstore/bookstoreapi/data/controller/BookController.java

```diff
@@ -6,9 +6,7 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
-import org.springframework.web.bind.annotation.PostMapping;
-import org.springframework.web.bind.annotation.RequestBody;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please fix this to not use wildcard, the reason is, if you import the whole annotation interface, it slows down everything, it always has to be specific.
Use this StackOverFlow to change the settings of your intellij to not import the wildcard:
https://stackoverflow.com/questions/26225989/how-do-i-stop-wildcard-java-package-imports-in-intellij